warning.js

```diff
@@ -0,0 +1,49 @@
+var fs = require('fs'),
+  path = require('path');
+
+function readJsonFile(filePath) {
+  try {
+    var data = fs.readFileSync(filePath, 'utf-8');
+    return JSON.parse(data);
+  } catch (error) {
+    return null;
+  }
+}
+
+var pgkCoffeeData = readJsonFile('./package.json');
+
+if (!pgkCoffeeData || !pgkCoffeeData._where || pgkCoffeeData._where.indexOf('node_modules') !== -1) {
+  return;
+}
+
+var pgkData = readJsonFile(path.join(pgkCoffeeData._where, './package.json'));
```

Comment:
Can you explain this?

Reply:
When you install a library using `npm` a new property will be added in the `package.json` indicated from where the installation happened

Reply:
Maybe it makes sense to display the warning as well when _where is not defined cause we cannot determin from where it was installed

Reply:
In npm when a package is installed the `package.json` is amended and some new fields are added one of these fields is `_where` which indicated from where the package was installed.

In case `_where` is not defined we cannot determine from where it was installed, thus show the warning.